app/views/organizations/foster_application_reviews/edit.html.erb

```diff
@@ -0,0 +1,61 @@
+<!--page heading-->
+<header class="pt-5 pb-5">
+  <div class="container">
+    <div class="text-center">
+      <h1 class="h4">
+        <%= @application.adopter_account.user.first_name %>
+        <%= @application.adopter_account.user.last_name %>'s application for
+        <%= @application.pet.name %>
+      </h1>
+    </div>
+  </div> <!--container-->
+</header>
+
+<!--main section-->
+<section class="pb-5" id="about_us">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-6 mx-auto card p-5 ">
+        <div class='d-flex flex-row-reverse'>
+          <%= link_to "#{@application.adopter_account.user.first_name}'s Profile", 
```

Comment:
```suggestion
          <%= active_link_to "#{@application.adopter_account.user.first_name}'s Profile", 
```
Let's use `active_link_to` in place of `link_to` so it knows to add the active class when we are on that page the link directs towards.

Reply:
Ah yes you mentioned that before. I will change that.